bsyncviewer/templates/faq.html

```diff
@@ -0,0 +1,114 @@
+{% extends 'base2.html' %}
+
+{% load staticfiles %}
+
+{% block title %}FAQ{% endblock title %}
+
+{% block content %}
+
+<div class="container bsync-main">
+
+	<h3 class="mat-h3">What is BuildingSync?</h3>
+	<p>BuildingSync is a standardized language for commercial building energy audit data that software developers can use to exchange data between audit tools. Taking the form of an <a href="https://en.wikipedia.org/wiki/XML_Schema_(W3C)" target="_blank" class="color-2 bold-link">XML schema</a>, BuildingSync can be required by building owners and audit program managers to allow data analysis and aggregation across multiple buildings in order to evaluate program performance and analyze trends.</p>
+
+	<p>It was developed using the standard energy data terminology defined in the Building Energy Data Exchange Specification (BEDES). BuildingSync is one of the first implementations of BEDES Version 1.0, and it was developed in close collaboration with the BEDES working groups.</p>
+
+	<p>BuildingSync standardizes the format of the energy audit data itself. It is not intended to standardize the calculations performed on these data by engineers, analysts, software packages, or other audit tools.</p>
+
+	<h3 class="mat-h3">What are the elements of BuildingSync?</h3>
+
+	<p>BuildingSync consists of two files and a reference sheet, which are all offered as free <a href="https://github.com/BuildingSync/schema/releases" target="_blank">downloads</a>. </p>
+
+
+	<section id="about" class="hero section-padding section-bord">
+		<div class="container">
+
+			<div class="row">
+				<!-- hero-item -->
+				<div class="hero-item col-md-4 text-center">
+					<span class="icon-clipboard"></span>
+					<div class="hero-item-content">
```

Comment:
this is baller!
<img width="1266" alt="image" src="https://user-images.githubusercontent.com/1907354/121899936-19a74580-cce2-11eb-8c39-80f2b51daab1.png">